src/common.rs

```diff
@@ -5,6 +5,10 @@
 
 #![allow(missing_docs)]
```

Comment:
It looks like this is here to avoid documenting the existing `pub const GAUSS_LEGENDRE_COEFFS`* values.
I added basically a stub documentation explaining the trait, and pointing to the std implementation for further docs.